base/libgit2/commit.jl

```diff
@@ -9,13 +9,16 @@ leading newlines removed). If `raw` is `true`, the message is not stripped
 of any such newlines.
 """
 function message(c::GitCommit, raw::Bool=false)
+    c_flag = Base.@_gc_preserve_begin c
```

Comment:
Can be written as 
```
@gc_preserve c begin

end
```
The underscore versions are only for early in the code base where that macro does not exist yet.